app/models/course/material/folder.rb

```diff
@@ -6,8 +6,9 @@ class Course::Material::Folder < ActiveRecord::Base
   before_destroy :destroy_children
 
   has_many :materials, inverse_of: :folder, dependent: :destroy, foreign_key: :folder_id,
-                       class_name: Course::Material.name
+                       class_name: Course::Material.name, autosave: true
```

Comment:
Can I spec this when this framework is integrated with other components ? I think it's not useful to spec it now, because this is the build-in function of rails.